pkg/globalnet/controllers/ipam/globalnet_egress.go

```diff
@@ -0,0 +1,32 @@
+package ipam
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/go-iptables/iptables"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/submariner/pkg/routeagent/controllers/route"
+)
+
+func (i *Controller) updateEgressRulesForPod(podIP, globalIP string, addOrDelete Operation) error {
+	ipt, err := iptables.New()
+	if err != nil {
+		return fmt.Errorf("error initializing iptables: %v", err)
+	}
+
+	ruleSpec := []string{"-p", "all", "-s", podIP, "-j", "SNAT", "--to", globalIP}
+	if addOrDelete == AddRules {
+		klog.V(4).Infof("Installing iptable egress rules for pod: %s", strings.Join(ruleSpec, " "))
```

Comment:
should this be V(6)? 4 might fill up logs pretty quick. Applies to all klog.V(4) logs. Can revisit later though.